downloads/platform.md

```diff
@@ -46,7 +46,7 @@ Julia installs all its files in a single directory. Deleting the directory where
 
 ## Windows
 
-Julia is available for Windows 7 and later for both 32-bit and 64-bit versions.
+The following lists instructions for supported and unsupported versions of Windows. See [supported platforms](/downloads/#supported_platforms) for supported Windows versions.
```

Comment:
I'm not a native English speaker, but "The following lists instructions..." sounds odd to me.  Maybe you can reword it in a less ambiguous way?